internal/models/refresh_token.go

```diff
@@ -103,18 +103,6 @@ func RevokeTokenFamily(tx *storage.Connection, token *RefreshToken) error {
 	return nil
 }
 
-func FindTokenBySessionID(tx *storage.Connection, sessionId *uuid.UUID) (*RefreshToken, error) {
```

Comment:
It's not the same implementation as FindCurrentlyActiveRefreshToken. Let's not modify right now.

Reply:
What's an example where they would differ? I'm aware that they aren't the same in terms of code but conceptually I would think that we should be replacing the active token.